.vscode/settings.json

```diff
@@ -5,5 +5,13 @@
     "autoDocstring.docstringFormat": "google",
     "autoDocstring.guessTypes": true,
     "autoDocstring.quoteStyle": "\"\"\"",
-    "autoDocstring.includeExtendedSummary": true
-}
+    "autoDocstring.includeExtendedSummary": true,
+    "python.testing.pytestArgs": [
```

Comment:
Nice! That is something we need. I tried to cover all possibilities but sometimes you oversee something. That is really helpful.